app-info/eos-extra.xml

```diff
@@ -1,20 +1,5 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Sorry, my last PR changed this process (again) and I forgot to update the README on how to do it. In your toolbox container, run `apt install gettext itstool make`. Then run `make` and commit all the changes. In addition to the `eos-extra.xml` file, it will update several files in `app-info/po`. Just let it do that. I think I need to poke at that part a bit more, but the changes are harmless.

I'm going to push a commit to master explaining this in the README.

Reply:
I expanded [the notes](https://github.com/endlessm/gnome-software-data/blob/master/app-info/README.md#how-to-add-a-progressive-web-application) to better describe what needs to be done.

Reply:
also weird cause my local copy is also lowercase...

Reply:
I don't think it matters but that is a little strange.